README.md

```diff
@@ -32,4 +32,8 @@ Open `settings.json` and add configs
 
 #### 2. Copy the command
 - Use key bindings `cmd+shift+c` for Mac or `ctrl+shift+c` for PC to copy the target command. Now you can paste the command in the terminal of your choice.
-![](sample.gif)
+
+<html>
+  /*![](sample.gif)*/
+<img src="sample.gif" height="600" width="750" align="center">
```

Comment:
Let's remove the comment and could you change the height to 500 and width to 600.

Reply:
sure

Reply:
did it.